tests/dummy/app/pods/size/controller.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
This is still present.

Reply:
which line you're referring to?

Reply:
https://github.com/quincyle/ember-frost-list/blob/84ccd52c5bafe328ed9c2c99cff5b01c9d43f4db/tests/dummy/app/pods/size/controller.js#L1-L3